video/out/gpu_next/context.c

```diff
@@ -88,7 +88,7 @@ static bool d3d11_pl_init(struct vo *vo, struct gpu_ctx *ctx,
     ctx->swapchain = pl_d3d11_create_swapchain(d3d11,
         pl_d3d11_swapchain_params(
             .swapchain = swapchain,
-            .disable_10bit_sdr = ra_d3d11_ctx_prefer_8bit_output_format(ctx->ra_ctx),
```

Comment:
I have added support for dither-depth=auto for d3d11, https://github.com/mpv-player/mpv/pull/13646 should be better than just hard coding this.

Can leave it for vulkan though.